third_party/python/cpplint/BUILD

```diff
@@ -7,7 +7,8 @@ load("@heron_py_deps//:requirements.bzl", "requirement")
 
 py_binary(
     name = "cpplint",
-    entrypoint = "cpplint",
+    srcs = ["empty.py"],
```

Comment:
@surahman / @nicknezis Wasn't exactly sure what to do here.  `py_binary` requires the `srcs` to have something in it.  So to get around this, I put an `empty.py` file in folder.

Reply:
Thanks @joshfischer1108 I will look through the files and see where else I need to do this. I am still learning Bazel at this point so @nicknezis if you have the time please feel free to jump in anywhere at any time.